pysysc/structural.py

```diff
@@ -94,10 +96,10 @@ def setup(log_level = logging.WARNING):
                 }
             cpp.sc_core.sc_report_handler.set_verbosity_level(verb_lut[log_level]);
         cpp.sc_core.sc_report_handler.set_actions(cpp.sc_core.SC_ID_MORE_THAN_ONE_SIGNAL_DRIVER_, cpp.sc_core.SC_DO_NOTHING);
-        try:
-            cpp.scc.init_cci("GlobalBroker")
-        except Exception:
-            pass
+        #try:
```

Comment:
Dead code.  If there is a reason to leave the commented block here, it should be documented with a comment.